docs/user_guide/hopsworks/hive.rst

```diff
@@ -14,11 +14,35 @@ This page serves as a guide on how to use Hive from within Hopsworks. For inform
 Using HopsHive with Hopsworks
 ------------------------------
 
-In order to use HopsHive from Hopsworks, users first need to create a database. Each project can have *at most one* Hive database, which is created when the Hive service is enabled for the project. Users can enable the Hive service either when creating a new project or via the *Settings* section.
+In order to use HopsHive from Hopsworks, users first need to create a database. Each project that is created with the Hive service enabled gets a Hive database. Users can enable the Hive service either when creating a new project or via the *Settings* section.
+
+.. _hive1.png: ../../_images/hive1.png
+.. figure:: ../../imgs/hive1.png
+    :alt: Enable the Hive service for a project
+    :target: `hive1.png`_
+    :scale: 100%
+    :align: center
+    :figclass: align-center
 
 Once the database is created, users are able to see in the *Datasets* view a new dataset called ``projectName.db``. This new dataset is the Hive database for the project and contains Hive's data.
 
-Users can then run queries using *Apache Zeppelin* (the Zeppelin service needs to be enabled for the project). To run a query, users need to create a notebook and use ``HopsHive`` as the interpreter. This can be done in either one of three ways:
+.. _hive2.png: ../../_images/hive2.png
+.. figure:: ../../imgs/hive2.png
+    :alt: Hive database dataset in Hopsworks
+    :target: `hive2.png`_
+    :scale: 100%
+    :align: center
+    :figclass: align-center
+
+There are two ways to run queries against Hive from Hopsworks:
+
+- Using *Apache Zeppelin* and the ``HopsHive`` interpreter
+- Using ``SparkSQL`` inside *Jobs* or *Jupyter Notebooks*
+
+Running Queries using Zeppelin
+----------------
+
+Users can  run queries using *Apache Zeppelin* (the Zeppelin service needs to be enabled for the project). To run a query, users need to create a notebook and use ``HopsHive`` as the interpreter. This can be done in either one of three ways:
```

Comment:
@Limmen I know it's not your wrongdoing since we haven't updated the docs for Zeppelin in a while, but it's a change to remove the `(the Zeppelin service needs to be enabled for the project)` :)

Reply:
@tkakantousis Yeah :) Will fix